handlers/handler.go

```diff
@@ -29,12 +36,17 @@ const SessionUsername = "username"
 // library along with their other standard init operations.
 var AllowOrigin = "*"
 
+// ErrRedirect indicates that the error intends to redirect the user to the proper spot.
+var ErrRedirect = errors.New("redirection")
+
 var (
 	errNoCapability  = errors.New("no capability to perform desired operation")
 	errInvalidCookie = errors.New("cookie was invalid")
 )
 
-var routeTransformer = regexp.MustCompile(`(?:{([^}]+)})+`)
+var (
+	routeTransformer = regexp.MustCompile(`(?:{([^}]+)})+`)
+)
```

Comment:
Not a big deal, but this file has both `var foo` and `var ( foo )` declarations... maybe it's worth making these consistent or perhaps grouping the vars from 37 - 49 into one `var ( ... )` block?

Reply:
I tend to group private and public ones, but not together (as golint requirements make it look gory). I'll review this and make sure it's honoring that rules

Reply:
oh I see what I did here; I grouped up the errors. Ok, I'll collate these.